bot.js

```diff
@@ -52,7 +57,7 @@ controller.hears(".*", ["direct_message","direct_mention","mention","ambient"],
 
       const commands = [
         quote(['say', '--output-file', 'out.aiff', '--voice', 'Daniel', text]),
-        'afplay --volume .7 --rate 1.2 --time 6 out.aiff',
+        `afplay --volume .7 --rate 1.2 --time ${maxSpeakingTimeInSeconds} out.aiff`,
```

Comment:
There's potential for shell injection here, but it would be coming from a user-supplied parameter so ehhh